orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -1113,6 +1113,34 @@ at::Tensor& mm_out(
   return out;
 }
 
+// aten::squeeze(Tensor(a) self) -> Tensor(a)
+at::Tensor squeeze(
+  const at::Tensor& self) {
+  ORT_LOG_FN(self);
+
+  if (
+    !IsSupportedType(self, {at::kBFloat16,at::kBool,at::kByte,at::kDouble,at::kFloat,at::kHalf,at::kInt,at::kLong,at::kShort}))
+    return at::native::call_fallback_fn<
+      &at::native::cpu_fallback,
+      ATEN_OP(squeeze)>::call(self);
+
+  auto& invoker = GetORTInvoker(self.device());
+
+  auto ort_input_0_self = create_ort_value(invoker, self);
+
+  std::vector<OrtValue> ort_outputs_0_Squeeze(1);
```

Comment:
can ort_input_0_self be passed in with this vector so that we can just return self directly?

Reply:
never mind